medium/_01_fixed/cases/fview1.sql

```diff
@@ -1,17 +1,17 @@
 autocommit off;
 call login('joe') on class db_user;
-create class xxx (a int);
-insert into xxx values(6);
-create vclass xxx_v (a1 int) as select a from xxx;
+create class joe.xxx (a int);
+insert into joe.xxx values(6);
+create vclass joe.xxx_v (a1 int) as select a from joe.xxx;
```

Comment:
No need to specify owner while creating.
Also, an owner does not need to be specified when using an owned object.
```
create class xxx (a int);
insert into xxx values(6);
create vclass xxx_v (a1 int) as select a from xxx;
```

Reply:
reflected. please check.

Reply:
Thank you.